tests/functional/object/mcg/ui/test_bucket_lifecycle_ui.py

```diff
@@ -0,0 +1,387 @@
+import logging
+import time
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1,
+    black_squad,
+    ui,
+)
+from ocs_ci.ocs.ui.page_objects.bucket_lifecycle_ui import BucketLifecycleUI
+from ocs_ci.ocs.ui.page_objects.buckets_tab import BucketsTab
+from ocs_ci.helpers.helpers import create_unique_resource_name
+
+logger = logging.getLogger(__name__)
+
+
+@black_squad
+@pytest.mark.polarion_id("OCS-6863")
```

Comment:
polarion IDs should be per test case. Here, wach test function is expected to have a unique polarion ID